test/visual/flex-child-tabs.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+  <meta charset="UTF-8">
+  <title>Flex child tabs visual tests</title>
+</head>
+
+<body>
+  <style media="screen">
+    .capture-block {
+      display: flex;
+      width: 400px;
+    }
+  </style>
+
+  <div class="capture-block" id="flex-child-tabs">
+    <vaadin-tabs>
+      <vaadin-tab>Foo</vaadin-tab>
+      <vaadin-tab>Bar</vaadin-tab>
```

Comment:
Nit: I would add one more `tab` and extend the width for consistency.

Reply:
Done.